op-alt-da/damock.go

```diff
@@ -130,6 +134,10 @@ func (s *FakeDAServer) HandleGet(w http.ResponseWriter, r *http.Request) {
 
 func (s *FakeDAServer) HandlePut(w http.ResponseWriter, r *http.Request) {
 	time.Sleep(s.putRequestLatency)
+	if s.failoverCount > 0 {
```

Comment:
Is it just to simplify testing?

Reply:
Because we only want to cause a temporary failure. So this way we can say "pretend you are down for the next failoverCount calls", and after that the system becomes normal again.